keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -980,8 +918,13 @@ class SqlArtifactRepository(
       val artifactVersions = deliveryConfig.artifacts.map { artifact ->
         val versionsInEnvironment = jooq
           .select(
-            ENVIRONMENT_ARTIFACT_VERSIONS.ARTIFACT_VERSION,
+            ARTIFACT_VERSIONS.NAME,
```

Comment:
@robfletcher This was the scariest function for me to change by far. I'd appreciate your attention here.

Reply:
Am I missing something? This function doesn't appear to be logically different.

Reply:
No, you're right. It's supposed to be functionally equivalent, it's just that it's pretty complex and so I was afraid I might fat-finger something, is all. Having said that, looking at the diff again now, it doesn't look that scary. Thanks for looking anyway!